src/tools/tidy/src/deps.rs

```diff
@@ -42,13 +42,16 @@ const EXCEPTIONS: &[(&str, &str)] = &[
     ("bitmaps", "MPL-2.0+"),                                 // cargo via im-rc
     ("fiat-crypto", "MIT OR Apache-2.0 OR BSD-1-Clause"),    // cargo via pasetors
     ("subtle", "BSD-3-Clause"),                              // cargo via pasetors
+    ("imara-diff", "Apache-2.0"),                            // cargo via gix
+    ("sha1_smol", "BSD-3-Clause"),                           // cargo via gix
+    ("unicode-bom", "Apache-2.0"),                           // cargo via gix
     ("instant", "BSD-3-Clause"), // rustc_driver/tracing-subscriber/parking_lot
     ("snap", "BSD-3-Clause"),    // rustc
     ("fluent-langneg", "Apache-2.0"), // rustc (fluent translations)
     ("self_cell", "Apache-2.0"), // rustc (fluent translations)
     // FIXME: this dependency violates the documentation comment above:
     ("fortanix-sgx-abi", "MPL-2.0"), // libstd but only for `sgx` target
-    ("dunce", "CC0-1.0"),            // cargo (dev dependency)
+    ("dunce", "CC0-1.0 OR MIT-0"),   // cargo (dev dependency)
```

Comment:
This is no longer just a dev-dependency, right?

Reply:
Hmm… I didn't see anything other than dev-deps.

```console
$ cargo tree -i dunce
dunce v1.0.3
└── snapbox v0.4.8
    └── cargo-test-support v0.1.0 (/Users/whlo/wd/cargo/crates/cargo-test-support)
        [dev-dependencies]
        └── cargo v0.70.0 (/Users/whlo/wd/cargo)
    [dev-dependencies]
    └── cargo v0.70.0 (/Users/whlo/wd/cargo)
```


It may be just their license changed.

- https://gitlab.com/kornelski/dunce/-/commit/87fe0a4acfc8fde3139830ff07db8cbd8155d035
- https://gitlab.com/kornelski/dunce/-/commit/e4057df8809de4c08dd40910329df084506bcf74

And the second one looks like they wanted to make it compatiable with rustc (which has yet published).

Reply:
Yea, but also this is now being used via gix, which does not appear to be a dev-dependency.

(The comments here aren't critical, but it seems that while we're updating this entry, it seems good to also update that.)

Reply:
I forgot to pass `--target all`. Rebased and pushed. Thanks for the catch!